apps/web/src/components/ProgressPreview/ProgressPreview.tsx

```diff
@@ -0,0 +1,37 @@
+'use client'
+
+import {Progress} from '@nanoui/core'
+import {type FC, useState} from 'react'
+import styles from './ProgressPreview.module.css'
+
+const ProgressPreview: FC = () => {
```

Comment:
```suggestion
export const ProgressPreview: FC = () => {
```

А в конце компонента export убрать

Reply:
d59b3b6547f27fdfde916dc5eb9fe96526f1e7ef